src/app/examples/02-multiple-selection-example/multiple-selection-example.component.html

```diff
@@ -2,8 +2,8 @@
 <h3>Multiple selection</h3>
 <p>
   <mat-form-field>
-    <mat-select [formControl]="bankMultiCtrl" placeholder="Banks" [multiple]="true" #multiSelect>
-      <ngx-mat-select-search [formControl]="bankMultiFilterCtrl"></ngx-mat-select-search>
+    <mat-select [formControl]="bankMultiCtrl" placeholder="Banks" [multiple]="true" #multiSelect disableOptionCentering>
+      <ngx-mat-select-search [showToggleAllCheckbox]="true" (toggleSelectAll)="toggleSelectAll($event)" [formControl]="bankMultiFilterCtrl"></ngx-mat-select-search>
```

Comment:
could you please duplicate the multiple-selection example, with and without toggle checkbox?

Reply:
within the same component? or just create a new component called multiple-selection-select-all-example ?

Reply:
please create a new component called multiple-selection-select-all-example.
this way we can test all features by checking the examples

Reply:
it has been added in the latest commit as 06- multiple-selection-select-all-example